gammapy/stats/feldman_cousins.py

```diff
@@ -391,7 +395,15 @@ def fc_find_average_upper_limit(x_bins, matrix, upper_limit, mu_bins):
     number_points = len(x_bins)
 
     for i in range(number_points):
-        limit = fc_find_limit(x_bins[i], upper_limit, mu_bins)
+        # Bins with very low probability will not contribute to average limit
+        if matrix[0][i] < prob_limit:
+            continue
+        try:
+            limit = fc_find_limit(x_bins[i], upper_limit, mu_bins)
+        except:
+            print("Warning: Calculation of average limit incomplete!")
+            print("Add more bins in mue direction or decrease prob_limit.")
```

Comment:
`mue` -> `mu`.

You should never print from library code, because then users (e.g. from pipelines) don't have easy control over the output. You can either emit a `Warning` or call `log.warning`.
Here's an explanation of how we call `log.warning` from Gammapy:
http://docs.gammapy.org/en/latest/development/howto.html#generating-log-messages